media/js/mozorg/contribute/signup.js

```diff
@@ -0,0 +1,36 @@
+var $tooltipTrigger = $('.tooltip-trigger');
```

Comment:
To keep from polluting the global namespace, can we wrap this code in an IIFE like we do elsewhere?